pkg/ingester/owned_streams.go

```diff
@@ -3,15 +3,16 @@ package ingester
 import (
 	"sync"
 
+	"github.com/grafana/dskit/services"
 	"go.uber.org/atomic"
 )
 
 type ownedStreamService struct {
-	tenantID   string
-	limiter    *Limiter
-	fixedLimit *atomic.Int32
+	services.Service
```

Comment:
If we keep this separated from `recalculateOwnedStreams` I suppose we don't need it anymore.

Reply:
yes, you are right